api/src/services/cards.service.ts

```diff
@@ -69,5 +102,14 @@ export class CardService {
 
     return createdPriceHistory;
   }
+
+  public async createCardRating(cardId : ObjectId | string , cardRatingData: CardRating): Promise<CardRating> {
+    const createdCardRating: CardRating = await CardRatingModel.create(cardRatingData);
+    
+    //TODO: add some kind of catch for catching errors
+    // if (!createdCardRating) throw new HttpException(409, "Card Rating doesn't exist");
```

Comment:
Noticed this comment, not sure if you wanted to added error handling.

Reply:
You're right. Looks like I had left the comment to do it. And I already added it into the code, but I had it commented out haha.